src/Pages/EmailList/TextEditor.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+
+import React from 'react';
+import { Editor } from '@tinymce/tinymce-react';
+export default function TextEditor(props) {
+  return (
+    <Editor
+      apiKey="3q3vjb0bbtg3jx437evqdsttkyz77mwas325effmyrxwrk33"
```

Comment:
Move this to the config file.

Reply:
Slight problem. this text editor file is in the src folder and I can't get the api key from the config.json file since it's outside of the src folder. I'll see if aash is available today and we'll try to figure it out. Currently some commented out stuff, but it's all WIP, so all that will be cleaned up.

Reply:
Haha jkkk there's a config.json in src